src/jit/codegen.h

```diff
@@ -464,7 +464,8 @@ protected :
                                     INDEBUG_LDISASM_COMMA(CORINFO_SIG_INFO* sigInfo)
                                     void*                 addr
                                     X86_ARG(ssize_t       argSize),
-                                    emitAttr              retSize,
+                                    emitAttr              retSize
+                                    FEATURE_UNIX_AMD64_STRUCT_PASSING_ONLY_ARG(emitAttr retSize1),
```

Comment:
Instead of having two overloads for CodeGen::genEmitCall() we should refactor it to be one call with a signature that takes as struct with fields:
void * addr;  // Direct Call
ssize_t argSize;  // X86
emitAttr retSize;
emitAttr retSize2; // UNIX_ABI
IL_OFFSET ilOffset;
regNumber base;
bool isJump;
bool isNoGC;
GenTreeIndir*  indir;  // Indirect call